erddap/content/datasets.xml

```diff
@@ -9411,4 +9411,434 @@ AZOMP's sampling scheme off Atlantic Canada and the Labrador Sea involves annual
   </dataset>
   <!-- DFO BIO Maritime Region Ecosystem Survey (MRES): END -->
 
+  <!-- DFO BIO Barrow Strait Program: START -->
+  <dataset type="EDDTableFromMultidimNcFiles" datasetID="bio_barrow_strait_program_rosette" active="true">
+    <reloadEveryNMinutes>10080</reloadEveryNMinutes>
+    <updateEveryNMillis>10000</updateEveryNMillis>
+    <fileDir>/datasets/dfo/bio/ctd/Barrow Strait Monitoring Program/</fileDir>
+    <fileNameRegex>.*\.ODF\.nc</fileNameRegex>
+    <recursive>true</recursive>
+    <pathRegex>.*</pathRegex>
+    <metadataFrom>last</metadataFrom>
+    <standardizeWhat>0</standardizeWhat>
+    <removeMVRows>true</removeMVRows>
+    <sortFilesBySourceNames></sortFilesBySourceNames>
+    <fileTableInMemory>false</fileTableInMemory>
+    <accessibleViaFiles>true</accessibleViaFiles>
+    <!-- Please specify the actual cdm_data_type (TimeSeries?) and related info below, for example...
+          <att name="cdm_timeseries_variables">station_id, longitude, latitude</att>
+          <att name="subsetVariables">station_id, longitude, latitude</att>
+      -->
+    <addAttributes>
+      <att name="organization">Fisheries and Oceans Canada</att>
+      <att name="organization_fr">Pêches et Océans Canada</att>
+      <att name="institution">DFO BIO</att>
+      <att name="cdm_data_type">Profile</att>
+      <att name="cdm_profile_variables">platform_name,chief_scientist,cruise_name,cruise_number,geographic_area,id,station,event_number,profile_direction,latitude,longitude,profile_start_time,profile_end_time</att>
+      <att name="Conventions">CF-1.6,CF-1.7,CF-1.8,ACDD-1.1,ACDD-1.3</att>
+      <att name="contributor_name">Null</att>
+      <att name="contributor_role">custodian,distributor,owner</att>
+      <att name="creator_name">Null</att>
+      <att name="creator_type">Null</att>
+      <att name="publisher_email">Null@null.ca</att>
+      <att name="publisher_institution">Fisheries and Oceans Canada</att>
+      <att name="publisher_name">Null</att>
+      <att name="publisher_url">https://www.dfo-mpo.gc.ca/index-eng.html</att>
+      <att name="description">Barrow Strait Program</att>
+      <att name="description_fra">MRES</att>
+      <att name="geospatial_lat_units">degrees_north</att>
+      <att name="geospatial_lon_units">degrees_north</att>
+      <att name="geospatial_vertical_positive">down</att>
+      <att name="geospatial_vertical_units">m</att>
+      <att name="header"></att>
+      <att name="infoUrl">https://www.dfo-mpo.gc.ca/science/data-donnees/azmp-pmza/index-eng.html</att>
+      <att name="keywords">CTD</att>
+      <att name="keywords_fra">CTD</att>
+      <att name="keywords_vocabulary">CIOOS Keywords</att>
+      <att name="license">Open Government Licence - Canada</att>
+      <att name="naming_authority">CF Standard Name Table v29</att>
+      <att name="creator_type">institution</att>
+      <att name="sourceUrl">(local files)</att>
+      <att name="platform">research vessel</att>
```

Comment:
could use a `<att name="platform_vocabulary">http://vocab.nerc.ac.uk/collection/L06/current/</att>`

Reply:
Oh yeah thanks @n-a-t-e I'll those to the future ODF datasets